e2e/testcafe-devextreme/tests/dataGrid/headerFilter/headerFilter.ts

```diff
@@ -10,6 +10,46 @@ fixture.disablePageReloads`Header Filter`
 
 const GRID_CONTAINER = '#container';
 
+test('Data should be filtered if (Blank) is selected in the header filter (T1257261)', async (t) => {
+  const result: string[] = [];
+  const dataGrid = new DataGrid(GRID_CONTAINER);
+  const headerCell = dataGrid.getHeaders().getHeaderRow(0).getHeaderCell(1);
+  const dataCell = dataGrid.getDataRow(0).getDataCell(0);
+  const filterIconElement = headerCell.getFilterIcon();
+  const headerFilter = new HeaderFilter();
+  const buttons = headerFilter.getButtons();
+  const list = headerFilter.getList();
+
+  await t.click(filterIconElement)
+    .click(list.getItem(1).element)
```

Comment:
I suggest adding comments about what items you select/unselect. For example:
`.click(list.getItem(1).element) // Select second item with value 'Item 1'`

Reply:
Good suggestion. Added.